src/server/test_utils.cc

```diff
@@ -659,8 +659,9 @@ void BaseFamilyTest::ExpectConditionWithinTimeout(const std::function<bool()>& c
 
 Fiber BaseFamilyTest::ExpectConditionWithSuspension(const std::function<bool()>& condition) {
   TransactionSuspension tx;
-  tx.Start();
-  auto fb = pp_->at(0)->LaunchFiber([condition, tx = std::move(tx)]() mutable {
+  pp_->at(0)->Await([&] { tx.Start(); });
```

Comment:
we require now access to `ServerStats` object so transactions must be schedules in a proactor thread.